<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowreclamation</class>
 <widget class="QDialog" name="windowreclamation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>971</width>
     <height>621</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>gestion de reclamations</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>951</width>
       <height>511</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>ajouter</string>
      </attribute>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>titre:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>180</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>corps:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>290</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id_personnel:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter_rec">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>390</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="idr_ajout">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="corps">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>170</y>
         <width>211</width>
         <height>87</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="id_per">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>290</y>
         <width>73</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="titre">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>941</width>
         <height>451</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0.338235, y1:0.006, x2:1, y2:1, stop:0.205882 rgba(141, 100, 200, 255), stop:1 rgba(112, 165, 255, 255));
background-color: qconicalgradient(cx:0.618, cy:0.829682, angle:75.2, stop:0 rgba(74, 94, 234, 255), stop:0.779412 rgba(124, 89, 220, 255));
border-color: rgb(255, 255, 255);
background-color: qlineargradient(spread:reflect, x1:0.859, y1:0.402591, x2:0.083, y2:0.142, stop:0.205882 rgba(129, 79, 200, 255), stop:0.877451 rgba(40, 104, 255, 255));
background-color: qlineargradient(spread:reflect, x1:0.609, y1:0.17, x2:0.083, y2:0.142, stop:0.205882 rgba(129, 79, 200, 255), stop:0.877451 rgba(40, 104, 255, 255));
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>label_1</zorder>
      <zorder>label_11</zorder>
      <zorder>label_13</zorder>
      <zorder>label_14</zorder>
      <zorder>label_15</zorder>
      <zorder>ajouter_rec</zorder>
      <zorder>idr_ajout</zorder>
      <zorder>corps</zorder>
      <zorder>id_per</zorder>
      <zorder>titre</zorder>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>supprimer</string>
      </attribute>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="supp_rec">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="supp_rec_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>130</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>911</width>
         <height>451</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0.338235, y1:0.006, x2:1, y2:1, stop:0.205882 rgba(141, 100, 200, 255), stop:1 rgba(112, 165, 255, 255));
background-color: qconicalgradient(cx:0.618, cy:0.829682, angle:75.2, stop:0 rgba(74, 94, 234, 255), stop:0.779412 rgba(124, 89, 220, 255));
border-color: rgb(255, 255, 255);
background-color: qlineargradient(spread:reflect, x1:0.859, y1:0.402591, x2:0.083, y2:0.142, stop:0.205882 rgba(129, 79, 200, 255), stop:0.877451 rgba(40, 104, 255, 255));
background-color: qlineargradient(spread:reflect, x1:0.609, y1:0.17, x2:0.083, y2:0.142, stop:0.205882 rgba(129, 79, 200, 255), stop:0.877451 rgba(40, 104, 255, 255));
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>label_2</zorder>
      <zorder>label_16</zorder>
      <zorder>supp_rec</zorder>
      <zorder>supp_rec_2</zorder>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>afficher</string>
      </attribute>
      <widget class="QTableView" name="tab_rec">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>60</y>
         <width>451</width>
         <height>271</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_id">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_nom">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_fonc">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_tel">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>170</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_mail">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>220</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_ville">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>270</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_ad">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>320</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_cin">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>20</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_sal">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>140</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_etat">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>190</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom et prenom:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>fonctionnalite:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>numero de telephone:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>216</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>email:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>275</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ville:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>320</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>adresse:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>20</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>cin:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>80</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>date:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>140</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>salaire:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>295</x>
         <y>195</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>etat:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_date">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="affichage_2">
       <property name="geometry">
        <rect>
         <x>850</x>
         <y>20</y>
         <width>81</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>afficher</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>931</width>
         <height>471</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0.338235, y1:0.006, x2:1, y2:1, stop:0.205882 rgba(141, 100, 200, 255), stop:1 rgba(112, 165, 255, 255));
background-color: qconicalgradient(cx:0.618, cy:0.829682, angle:75.2, stop:0 rgba(74, 94, 234, 255), stop:0.779412 rgba(124, 89, 220, 255));
border-color: rgb(255, 255, 255);
background-color: qlineargradient(spread:reflect, x1:0.859, y1:0.402591, x2:0.083, y2:0.142, stop:0.205882 rgba(129, 79, 200, 255), stop:0.877451 rgba(40, 104, 255, 255));
background-color: qlineargradient(spread:reflect, x1:0.609, y1:0.17, x2:0.083, y2:0.142, stop:0.205882 rgba(129, 79, 200, 255), stop:0.877451 rgba(40, 104, 255, 255));
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>label_28</zorder>
      <zorder>tab_rec</zorder>
      <zorder>label_17</zorder>
      <zorder>text_id</zorder>
      <zorder>text_nom</zorder>
      <zorder>text_fonc</zorder>
      <zorder>text_tel</zorder>
      <zorder>text_mail</zorder>
      <zorder>text_ville</zorder>
      <zorder>text_ad</zorder>
      <zorder>text_cin</zorder>
      <zorder>text_sal</zorder>
      <zorder>text_etat</zorder>
      <zorder>label_18</zorder>
      <zorder>label_19</zorder>
      <zorder>label_20</zorder>
      <zorder>label_21</zorder>
      <zorder>label_22</zorder>
      <zorder>label_23</zorder>
      <zorder>label_24</zorder>
      <zorder>label_25</zorder>
      <zorder>label_26</zorder>
      <zorder>label_27</zorder>
      <zorder>text_date</zorder>
      <zorder>affichage_2</zorder>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
